src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -0,0 +1,223 @@
+package net.maizegenetics.phgv2.api
+
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.vcf.VCFFileReader
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.Channel
+import net.maizegenetics.phgv2.utils.AltHeaderMetaData
+import net.maizegenetics.phgv2.utils.parseALTHeader
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.util.*
+
```

Comment:
Should have a header comment at the beginning of each class.

Reply:
Ok, I added this